src/styles/bloks/card.scss

```diff
@@ -0,0 +1,100 @@
+@use '../utils/variables' as *;
+
+body {
+  margin: 0;
+}
+
+.card {
+  width: $card-width;
+  height: $card-height;
```

Comment:
The variables `$card-width` and `$card-height` are used here, but it's important to ensure they are defined in the `variables` file being imported. If they are not defined, this will cause a compilation error.